languages/ruby/exercises/concept/instance-variables/.meta/design.md

```diff
@@ -0,0 +1,38 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept instance variables and "nil" in Ruby.
```

Comment:
I am not sure why "Concept" has been capitalized everywhere.  But I believe English doesn't generally work this way.

the concept "Instance Variables" might be more indicative that it is the concept that we are trying to teach.  The "nil" is what reminded me of this, with the quotes.  Because as a concept, it wouldn't be `nil` but "nil" as presented here, and so I think it is consistent to also quote the "instance variables" as the concept.

Reply:
I think it was once and I copy and pasted it. You're right, English doesn't work that way.

Reply:
```suggestion
The goal of this exercise is to teach the student the basics of the concept instance variables and "nil" in Ruby.
```